src/FV/src/math/lu_implementation.F90

```diff
@@ -1,18 +1,10 @@
-!! category: Morfeus-FV
-!! summary: Procedures for matrix LU factorization and solution
-!!
-!! Copyright Notice
-!! ----------------
-!!
-!!     (c) 2019 Guide Star Engineering, LLC
-!!     This Software was developed for the US Nuclear Regulatory Commission (US NRC)
-!!     under contract
-!!     "Multi-Dimensional Physics Implementation into Fuel Analysis under Steady-state and Transients (FAST)",
-!!     contract # NRC-HQ-60-17-C-0007
-!!
-
-
-
+!
+!     (c) 2019 Guide Star Engineering, LLC
+!     This Software was developed for the US Nuclear Regulatory Commission (US NRC)
+!     under contract "Multi-Dimensional Physics Implementation into Fuel Analysis under
+!     Steady-state and Transients (FAST)", contract # NRC-HQ-60-17-C-0007
```

Comment:
Changing this back has two problems:

1. It causes breakage with FORD when a pre-processed `.F90` file has a comment with single or double quotes separated by a line break
2. The change causes the copyright data to render nicely in FORD documentation. See https://sourceryinstitute.github.io/MORFEUS-Source/sourcefile/lu_implementation.f90.html for an example

Is there a reason you want to change it back? This contains the same information...